### PR TITLE
Preprod: movie: The 6th Day (2000) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Unoriginal, but Schwarzenegger's in it. Rating 3/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -170,7 +170,65 @@ asteroid impact [Sherman Klump had a dream in which a giant asteroid collided wi
...
+husband and wife [Central to the story were Adam and Natalie Gibson. Dr. Griffin Weir and his terminally ill clone wife Katherine.]
+unethical business practices [Replacement Technologies was secretly cloning people left, right and center in flagrant violation of the human cloning prohibiting "Sixth Day" laws.]
+what if I met my double [The Adam clone teamed up with the real article in order to rescue Adam's wife and daughter from the clutches of a ruthless billionaire CEO.]
+the quest for immortality [The CEO of Replacement Technologies Michael Drucker was engaged in nefarious research into human cloning because he wanted to conquer death; in particular his own.]
+familial love [The viewer is shown just how much both the Adam clone and the real article loved Adam's wife and young daughter.]
+human self improvement [The story culminated with both Adam and the Adam clone changing their view on cloning in general, and human cloning in particular. They went from thinking clones were weird and creepy, to accepting that clones were people too.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:214]: This doesn't sound right. The comment suggests that the Themer thinks the characters improved. But the theme in question is supposed to be about the characters thinking about themselves and trying to improve.

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -170,7 +170,65 @@ asteroid impact [Sherman Klump had a dream in which a giant asteroid collided wi
...
+virtual lover [Hank had a holographic, virtual "girlfriend".]
+the soul [The Adam clone wondered aloud about whether or not he had a soul.]
+hologram [The story is set in a near future when holographic technology was pervasive in society. Hank, for example, kept a non-sentient holographic "girlfriend" cooped up in his apartment.]
+friendship [Adam and Hank were best friends.]
+speculative weapon [The Replacement Technologies henchmen made frequent use of futuristic, directed energy handguns.]
+romantic love [Dr. Griffin Weir proclaimed his love to his cloned wife, Katherine, while she lay on her deathbed.]
+coping with a terminal illness [The Katherine clone was dying from cystic fibrosis.]
+coping with the death of a spouse [Dr. Griffin Weir confronted the billionaire CEO Michael Drucker for having seeded Weir's clone wife with the cystic fibrosis that led to her premature death.]
+pride goes before a fall [The billionaire CEO Michael Drucker uttered "I'm just taking over where God left off" in reference to his plan to secure his immortality via human cloning not long before he fell through a glass rooftop to his horrible death.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:228]: A case of hubris perhaps?

**Review by odinlake — COMMENTED** [review]
Comments inline

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Fair enough. I've deleted it - although future themers may dig up proper justification for it.

Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Note: I added **playing God with nature**.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:228]: Added!

**Review by odinlake — APPROVED** [review]

**Pull request merged**